pkg/lightning/backend/local/local.go

```diff
@@ -1381,6 +1383,27 @@ func (local *local) WriteToTiKV(
 	region *split.RegionInfo,
 	start, end []byte,
 ) ([]*sst.SSTMeta, Range, rangeStats, error) {
+	for _, peer := range region.Region.GetPeers() {
+		var e error
+		for i := 0; i < maxRetryTimes; i++ {
+			store, err := local.pdCtl.GetStoreInfo(ctx, peer.StoreId)
+			if err != nil {
+				e = err
+				continue
+			}
+			if store.Status.Capacity > 0 {
+				// The available disk percent of TiKV
+				ratio := store.Status.Available * 100 / store.Status.Capacity
+				if ratio < 10 {
+					return nil, Range{}, rangeStats{}, errors.Errorf("The available disk of TiKV only left %d, and capacity is %d", store.Status.Available, store.Status.Capacity)
```

Comment:
1. include the TiKV address or store ID or any identifying information into the error message
2. include the unit. i guess it is "bytes" in both cases
3. i think measuring the absolute amount of available space is better than a percentage. if the TiKV disk is very small, 10% avail is probably not enough; and if the TiKV is very large, 10% avail would be excessive.

Reply:
@Little-Wallace item 1 is still not fixed.